background/basicauth.js

```diff
@@ -17,37 +17,38 @@
  * along with Keywi.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-(function() {
+(async () => {
```

Comment:
Not sure how IIFE with async works, I guess unlike python asyncio this properly executes still since it's just a regular Promise automatically bound to the event loop